runtime/bindings/python/src/compatibility/pyngraph/node_factory.cpp

```diff
@@ -51,6 +52,19 @@ class NodeFactory {
         return op_node;
     }
 
+    std::shared_ptr<ngraph::Node> create(const std::string op_type_name) {
+        std::shared_ptr<ngraph::Node> op_node = std::shared_ptr<ngraph::Node>(m_opset.create(op_type_name));
+
+        NGRAPH_CHECK(op_node != nullptr, "Couldn't create operator: ", op_type_name);
+        NGRAPH_CHECK(!ngraph::op::is_constant(op_node),
+                     "Currently NodeFactory doesn't support Constant node: ",
+                     op_type_name);
+
+        NGRAPH_WARN << "Empty op created! Please assign inputs and attributes and run validate() before op is used.";
```

Comment:
Can you somehow also put a test to check if this error is raised correctly?

Reply:
This is not an error, it is only a warning for an empty op.